client/src/components/DataEntry/BoardTable.tsx

```diff
@@ -171,13 +178,14 @@ const BoardTable: React.FC<IProps> = ({ boardName, ballots, url }: IProps) => {
         columnWidths={cols}
         enableRowHeader={false}
       >
+        <Column key="container" name="Container" cellRenderer={renderCell} />
+        <Column key="tabulator" name="Tabulator" cellRenderer={renderCell} />
```

Comment:
should we make these two columns conditional? i.e. only show them when there is data for them? (if we have data in one of these fields for one ballot, we are guaranteed to have it in all ballots)

Reply:
@jonahkagan Oh yeah, I was going to ask specifically that but forgot, haha. If you have it in one field it's in all fields that would be relatively easy I think, yeah.